sysutils/salt/Portfile

```diff
@@ -23,56 +23,95 @@ if {$subport eq $name} {
     github.setup            saltstack salt ${version} v
     revision                1
 
-    python.versions         27 35 36
+    python.versions         35 36 37
```

Comment:
this doesn't do anything since the port name doesn't start with `py-`; so you can as well just remove this line